src/main/java/io/apicurio/datamodels/deref/DereferenceConstants.java

```diff
@@ -0,0 +1,12 @@
+package io.apicurio.datamodels.deref;
+
+public class DereferenceConstants {
```

Comment:
Can have a private constructor

Reply:
I never understood really why.  I mean, sure it means I cannot instantiate the class.  But who cares?  :)  If you want to create an instance of `DereferenceConstants` who am I to tell you you cannot?  lol